lua/zen-mode/plugins.lua

```diff
@@ -47,6 +47,16 @@ function M.kitty(state, disable, opts)
   vim.cmd([[redraw]])
 end
 
+function M.wezterm(state, disable, opts)
+  local stdout = vim.loop.new_tty(1, false)
+  if disable then
+    stdout:write(('\x1b]1337;SetUserVar=%s=%s\b'):format('ZEN_MODE', vim.fn.system({ 'base64' }, tostring(opts.font))))
```

Comment:
can this be modified to handle being in tmux too?

Reply:
I am not sure what you mean? The tmux integration is done separately in another file

Reply:
I was talking about the situations where you are running tmux within wezterm. I believe there are additional bits of ansi you need to add. If you don’t care to support that, that’s fine, it’s be nice though! 😁🙏🏻

Reply:
I do not currently have the time to dig into this, sorry. I suggest merging this as is and add this in the future (if someone needs it and knows how to/has the time to make this work).